### DISCUSSION TITLE
[select] select组件多选时，聚焦和失焦方法触发时机

### DISCUSSION THREAD

**Discussion by ykxbc** [element-plus/element-plus#15077] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `select`

## Description

多选情况下，change的时候(点击options)会触发失焦和聚焦事件
虽然是说了“当 input 失去/获取焦点时触发”，但是这种应用场景应该不多吧，感觉还是按照正常的失去/获取焦点来触发比较好（按照&__wrapper是否.is-focus来触发这两个事件）

### Additional comments

arco-design是正常的

**Comment by tolking** [comment]
我不是很明白你说的正确的焦点触发应该是什么样的流程。

但现在 ElSelect 是当整个组件聚焦时触发 focus，整个组件失焦时触发 blur
